run.py

```diff
@@ -320,6 +384,11 @@ def get_subject_code(fw_client, analysis_id):
 
         # Get config options
         config = gear_context.config
+        # Get config.json
```

Comment:
Given the use of gear context, do we need to load data from `config.json` into `config_json`?

Reply:
The issue is that there does not appear to be a way to get all inputs regardless of key, given the specificity of gear_context.get_input(). I could use keys specific to siena, but when I was writing the function, my hope was for it to be adaptable to most gears,  regardless of inputs